packages/adblocker/src/engine/engine.ts

```diff
@@ -229,6 +229,77 @@ export default class FilterEngine extends EventEmitter<EngineEventHandlers> {
     return engine as InstanceType<T>;
   }
 
+  public static fromEngines<T extends FilterEngine>(
+    this: new (...args: any[]) => T,
+    ...engines: T[]
+  ): T {
+    if (engines.length === 0) {
```

Comment:
I think it is better to throw as otherwise the config is empty

Reply:
Fixed by [`f194b98` (#4138)](https://github.com/ghostery/adblocker/pull/4138/commits/f194b98820a93ab5c48dd81a5cf41d3ef5b18796)